Fixes for Historic Frontend

kevindew opened the pull request:
This:

- adds a breadcrumb to take you to the current document
- changes the "Choose" copy to be "Updates"
- wraps all the diffs in wrapper elements so styles are correctly displayed

And actually a bunch of other stuff like
- New compare navbar
- Change links on the choose page
- Help text

kevindew committed:
Add breadcrumb to return to document

kevindew committed:
Show updates rather than a choose prompt

kevindew committed:
Wrap diffs in div.diff-checker div.diff-output

benjamin-mortimer reviewed (commented):
What's the logic of the change to 'document history'?

kevindew commented:
@benjamin-mortimer oh shoot that should have been a separate commit.

I feel that it's slightly clearer that content history as it's more precise, whereas content is a bit more ambiguous.

thomasleese commented on historical-frontend/choose.html:
```diff
@@ -5,7 +5,7 @@
 
 {% include_relative _heading.html %}
 
-<h1 class="heading-medium">Choose a version to start with</h1>
```
Hmm, I prefer this wording as it tells users what to do with the following links, otherwise it's just a list of links.

kevindew replied:
I think having a title that specifies that you have to do something is quite a strange pattern for GOV.UK* - To be honest I thought it was a hold over from when we had a more transactional flow. I'll have another look at it when I've cleared some stuff.

* I appreciate the irony of this given how the diff view has no grounding in existing patterns.

benjamin-mortimer commented:
@kevindew I think 'page history' would be a better version if that won't raise technical ambiguities. The user's come here from what in their mind is a 'webpage' so it makes sense to them to see a history of that 'page'. Understanding that a webpage is actually a 'document' is another cognitive step.

thomasleese commented:
@kevindew As part of this PR, we need to swap which edition is on the left and which is on the right in  the links on the start page.

kevindew commented:
@benjamin-mortimer sure, that sounds a good point. I'll try that out.

kevindew committed:
Add new change interface

This has a HTML dropdown for selecting changes

kevindew committed:
Include information about what is currently on GOV.UK

kevindew committed:
Update the change links to show diffs

kevindew committed:
Use page history instead of document history

kevindew committed:
Add some help content

kevindew committed:
A few small fixes

- Remove unnecessary CSS
- Use vertical-align: top when comparing lines
- Remove erroneous strong tags

kevindew commented:
As it's late I'm going to merge this in without review. But happy to discuss bits after merge as I've kind of steamrollered a few things in.
